internal/backend/basesvc/svc.go

```diff
@@ -162,9 +162,20 @@ func makeFxOpts(cfg *Config, opts RunOptions) []fx.Option {
 			mux.Handle("/favicon-32x32.png", srv)
 			mux.Handle("/favicon-16x16.png", srv)
 		}),
-		fx.Invoke(func(lc fx.Lifecycle, z *zap.Logger, httpsvc httpsvc.Svc) {
+		fx.Invoke(func(mux *http.ServeMux, tclient temporalclient.Client) {
+			_, uiAddr := tclient.TemporalAddr()
+
+			if uiAddr != "" {
+				mux.HandleFunc("/temporal", func(w http.ResponseWriter, r *http.Request) {
+					http.Redirect(w, r, uiAddr, http.StatusFound)
+				})
+			}
```

Comment:
NIT, remove indentation, reverse the if condition
```suggestion
			if uiAddr == "" {
			   return 
			}
		        mux.HandleFunc("/temporal", func(w http.ResponseWriter, r *http.Request) {
				http.Redirect(w, r, uiAddr, http.StatusFound)
			})
			
```

Reply:
done